container/Dockerfile.vllm

```diff
@@ -330,6 +273,8 @@ RUN --mount=type=secret,id=aws-key-id,env=AWS_ACCESS_KEY_ID \
         export CMAKE_CXX_COMPILER_LAUNCHER="sccache" && \
         export RUSTC_WRAPPER="sccache"; \
     fi && \
+    export CC=$(which gcc) && \
+    export CXX=$(which g++) && \
```

Comment:
I see lots of these changes across the files. I'm not sure you intended to change these lines but maybe caused by merge conflicts?

Reply:
`CC` and `CXX` are explicitly exported to force the build tools (like `maturin`) to use GCC. For my understanding, this can prevent edge cases where the environment might default to a different compiler (like `Clang`) and break the build.